src/index.ts

```diff
@@ -277,7 +281,7 @@ const extension: JupyterFrontEndPlugin<void> = {
       label: args => {
         const format = (args['format'] as ArchiveFormat) || '';
         const label = format.replace('.', ' ').toLocaleUpperCase();
-        return label ? `${label} Archive` : 'Download as an Archive';
+        return label ? `${label} Archive` : _trans.__('Download as an Archive');
```

Comment:
You can set a parametrize string as translatable using the following syntax:

```suggestion
        return label ? trans.__('%1 Archive', label) : trans.__('Download as an Archive');
```